site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/converter/OrganizationConverter.java

```diff
@@ -33,6 +33,17 @@
         if (input.getSiteType() != null) {
             fhirOrganization.setImplicitRules(input.getSiteType());
         }
+
+        if (input.getAddress() != null) {
+            fhirOrganization.addAddress().addLine(input.getAddress().getAddress1())
+                    .addLine(input.getAddress().getAddress2())
+                    .addLine(input.getAddress().getAddress3())
+                    .addLine(input.getAddress().getAddress4())
+                    .addLine(input.getAddress().getAddress5())
+                    .setCity(input.getAddress().getCity())
+                    .setCountry(input.getAddress().getCountry())
+                    .setPostalCode(input.getAddress().getPostcode());
+        }
```

Comment:
Should we have an `EntityConverter <Address>` instead of burying the mapping to FHIR Address in here? Can reuse it then.

Reply:
Agree if possible

Reply:
I have added EntityConverters for SiteAddressConverter (MTS Model) and SiteConverter (FHIR)